pipeline_blocks.py

```diff
@@ -140,40 +140,89 @@ def classifier_sklearn(sklearn_features, ClassifierClass, full_config, clf_name,
 def feature_extraction(config, train_mode, **kwargs):
```

Comment:
@pknut @jakubczakon I think that we need `feature_extraction` refactor. Now, User cannot try multiple models with fewer features. In `solution-1` we had 122 features, in `solution-2` we have 2.5k features, and here we are adding even more. IMHO it should be parametrizable what freature-sets User want to use in their training. For example, pick only basic_features and bureau features.